src/app/clusters/basic-information/BasicInformationCluster.cpp

```diff
@@ -204,12 +204,15 @@ inline CHIP_ERROR ReadCapabilityMinima(AttributeValueEncoder & aEncoder)
     return aEncoder.Encode(capabilityMinima);
 }
 
-inline CHIP_ERROR ReadConfigurationVersion(DeviceLayer::ConfigurationManager & configManager, AttributeValueEncoder & aEncoder)
-{
-    uint32_t configurationVersion = 0;
-    ReturnErrorOnFailure(configManager.GetConfigurationVersion(configurationVersion));
-    return aEncoder.Encode(configurationVersion);
-}
+// WORKAROUND: The ConfigurationVersion attribute is marked as mandatory and cannot be disabled, but it is provisional and
```

Comment:
Please add the task ID to not forget about it.

Reply:
Discussed f2f, it will be solved with the next upmerge